packages/edit-site/src/components/dataviews/dataviews.js

```diff
@@ -28,18 +28,63 @@ import TextFilter from './text-filter';
 export default function DataViews( {
 	data,
 	fields,
+	view,
+	onChangeView,
 	isLoading,
 	paginationInfo,
-	options,
+	options: { pageCount },
 } ) {
 	const dataView = useReactTable( {
 		data,
 		columns: fields,
-		...options,
+		manualSorting: true,
```

Comment:
For now we can add these internally(`manualSorting` and the rest manual props), but if we feed the component with static data, these should not be set to `true` values.

Reply:
Yeah, I'd like for these to remain internal implementation detail, we may have a way to check whether the data is static or not depending on the "data" props we provide later.